epam_task_02/task_02/my_text_reader/src/main/java/Runner.java

```diff
@@ -0,0 +1,31 @@
+package main.java;
+
+import main.java.entity.TextComponent;
+import main.java.entity.TextComposite;
+
+import java.io.*;
+
+public class Runner {
+    public static void main(String[] args) {
+        Reader reader;
+        String textFromFile = null;
+        try {
+            reader = new BufferedReader(new FileReader("src/resource/text.txt"));
+            StringBuffer stringBuffer = new StringBuffer();
```

Comment:
StringBuilder suits better here and in all other occurrences in code since it is not multi threading.